.gitignore

```diff
@@ -3,4 +3,6 @@
 CMakeCache.txt
 CMakeFiles/
 build/
-.vscode/
\ No newline at end of file
+.vscode/
+cmake-build-*/
```

Comment:
What is this for?

Reply:
Those are build files created by CLion. '*' is selected profile in the IDE.